src/dodal/devices/i04/transfocator.py

```diff
@@ -85,3 +85,8 @@ async def set(self, value: float):
                 self.beamsize_set_microns.set(value),
                 self._observe_beamsize_microns(),
             )
+        number_filters_rbv = await self.number_filters_sp.get_value()
+        vertical_lens_size_rbv = await self.vertical_lens_rbv.get_value()
```

Comment:
Better to put these in an `asyncio.gather`

Reply:
Oh yeah, definitely